### PR TITLE
Master pr

### PR DESCRIPTION
**Pull request by TheVeggydude**
v .4 of PyDash

### CODE REVIEW

File: build_and_run.sh [hunk 1]
```diff
@@ -1,5 +1,14 @@
 #!/bin/bash
 
+ExportEnvironmentVars()
+{
+  export FLASK_APP=pydash.py
+  export FLASK_ENV=development
+  export MAIL_USERNAME=noreply.pydashtestmail@gmail.com
+  export MAIL_PASSWORD=verysecurepydashpassword
```
Comment (FlyingBird95) [inline build_and_run.sh]: Don't put the password in the run-file

File: pydash-front/src/authenticated_app/dashboard/statistics_page/EndpointExecutionTimesPanel.js [hunk 2]
```diff
@@ -103,15 +26,15 @@ class EndpointExecutionTimesPanel extends Component {
     }
 
     componentDidMount() {
-        // TODO Once back-end has proper API endpoints, use that one instead of the overall one.
         axios(window.api_path + '/api/dashboards/' + this.props.dashboard_id, {
             method: 'get',
             withCredentials: true
         }).then((response) => {
+            console.log("Returned data", response);
```
Comment (FlyingBird95) [inline pydash-front/src/authenticated_app/dashboard/statistics_page/EndpointExecutionTimesPanel.js]: Debug line

File: pydash-front/src/authenticated_app/settings/SettingsPage.js [hunk 3]
```diff
@@ -118,31 +135,142 @@ handleDelete = (e) => {
...
-      console.log(response);
-      return <Redirect to="/" />
-  }).catch((error) => {
-      console.log('changing password failed');
-      this.handleClose();
-  });
+
+
+
+// handleSounds = (e) => {
```
Comment (FlyingBird95) [inline pydash-front/src/authenticated_app/settings/SettingsPage.js]: remove commented  code

File: build_and_run.sh (hunk 1 above)
Comment (InputUsername) [reply build_and_run.sh]: This is temporary, the deployed version will use secret credentials.

Comment (InputUsername) [reply build_and_run.sh]: (also see #421)

**Pull request merged**